tasks/ceph_deploy.py

```diff
@@ -239,14 +239,19 @@ def execute_ceph_deploy(cmd):
                         ceph_admin, conf_path, lines, sudo=True)
 
         # install ceph
-        install_nodes = './ceph-deploy install ' + \
-            (ceph_branch if ceph_branch else "--dev=master") + " " + all_nodes
+        ceph_sha = ctx.config['sha1']
+        devcommit = '--dev-commit={sha}'.format(sha=ceph_sha)
+        if ceph_branch:
+            option = ceph_branch
```

Comment:
`devcommit` is being constructed with `--dev-commit` but `ceph_branch` isn't. Shouldn't this line be:

    option = '--dev={branch}'.format(branch=ceph_branch)